.github/main.workflow

```diff
@@ -5,7 +5,7 @@ workflow "Pull Request Updates" {
 
 action "pr-filter-sync" {
     uses = "actions/bin/filter@master"
-    args = "action 'opened|review_requested|synchronize'"
```

Comment:
`review_requested` was adding in here due to GitHub Actions errors previously, but maybe those are fixed now. 🤞

Reply:
👍